test-case/check-pause-release-suspend-resume.sh

```diff
@@ -52,13 +52,14 @@ set -e
 ##  * No unexpected errors should be present in dmesg during or after test
 ##      completion.
 
-source $(dirname ${BASH_SOURCE[0]})/../case-lib/lib.sh
+CASEDIR=$(dirname "${BASH_SOURCE[0]}")
+source $CASEDIR/../case-lib/lib.sh
 
 OPT_NAME['m']='mode'         OPT_DESC['m']='test mode. Example: playback; capture'
 OPT_HAS_ARG['m']=1             OPT_VAL['m']='playback'
 
-OPT_NAME['p']='pcm'          OPT_DESC['p']='run test case on specified pipelines'
-OPT_HAS_ARG['p']=1             OPT_VAL['p']='id:any'
+OPT_NAME['S']='pcm'          OPT_DESC['S']='run test case on specified pipelines'
+OPT_HAS_ARG['S']=1             OPT_VAL['S']='id:any'
```

Comment:
You cannot keep the same help message for a new option that works very differently! Maybe: "run test case on pipelines matching the filter argument"

Reply:
Maybe you can refer to check-pause-resume.sh to keep your options consistent with it.

Reply:
@marc-hb The -S is used to specify the same PCM ID parameter as -p, so I copy the the help info. I change the option from -p to -S to align with case check-pause-resume.sh that uses -S to specify PCM ID.